scripts/ledger.lic

```diff
@@ -269,6 +298,22 @@ module Ledger
           silver = match[:silver].delete(',').to_i
           echo "recorded.deposit : #{silver}"
           self.record_transaction(amount: silver, type: "silver")
+        elsif (match = line.match(@withdraw_notes_regex))
+          silver = match[:silver].delete(',').to_i
+          fee = match[:fee].delete(',').to_i
+          silver += fee
+          echo "recorded.withdraw : #{silver}"
+          self.record_transaction(amount: -silver, type: "silver")
+        elsif (match = xml_line.match(@withdraw_notes_xml_regex))
+          fee = match[:fee].delete(',').to_i
+          silver = Ledger::Utility.get_note_value(match[:noteId]) # If further notes are withdrawn before this returns they will be missed

```

Comment:
Mentioned in opening comment but during this note read any further note withdraws will be missed.

Reply:
Have you thought about instead of reading the note to check the upstream buffer to see what the most recent WITHDRAW command was? Granted, that could get complex considering I'm not sure if people do w 100000 or what have you at the bank or what method to try to match against.